internal/command/fn/delete.go

```diff
@@ -12,4 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-some code
\ No newline at end of file
+package fn
+
+import (
+	"context"
+
+	"github.com/funlessdev/fl-cli/pkg/client"
+	"github.com/funlessdev/fl-cli/pkg/log"
+)
+
+type Delete struct {
+	Name      string `arg:"" name:"name" help:"name of the function to delete"`
```

Comment:
When deleting a function, the name arg should be required.

Reply:
I think args in kong are required by default (https://github.com/alecthomas/kong#supported-tags); I tried running `fn delete` without a name and it returned `error: expected "<name>"`

Reply:
Perfect, then we can resolve these conversations.